trunk/src/app/srs_app_gb28181.cpp

```diff
@@ -70,11 +70,15 @@ std::string srs_sip_state(SrsGbSipState ostate, SrsGbSipState state)
     return srs_fmt("%s->%s", srs_gb_sip_state(ostate).c_str(), srs_gb_sip_state(state).c_str());
 }
 
-SrsLazyGbSession::SrsLazyGbSession(SrsLazyObjectWrapper<SrsLazyGbSession>* wrapper_root)
+SrsLazyGbSession::SrsLazyGbSession() : sip_(NULL), media_(NULL)
```

Comment:
I would always prefer initialize the variable in the initialization list, the benefit is to avoid to invoke constructor, assign constructor and destructor again in this constructor body. (line 79 & 80).
```suggestion
SrsLazyGbSession::SrsLazyGbSession() : sip_(new SrsLazyGbSipTcpConn), media_(new SrsLazyGbMediaTcpConn())
```

of course remove line 79 & 80.

Reply:
Fixed.